### PR TITLE
Page with all required files

### PR DESCRIPTION
**Pull request by TheNightdrivingAvenger**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
```
Comment (TheNightdrivingAvenger) [inline index.html:4]: https://www.w3schools.com/html/html_responsive.asp

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+					<li><a href="PremiumPSD.html">PREMIUM PSD FILES</a></li>
+					<li><a href="About.html">ABOUT</a></li>
+					<li><a href="Blog.html">BLOG</a></li>
+				</ul>
+				<i class="fas fa-bars"></i>
+			</div>
+		</nav>
+        <div class="top-banner">
+			<a href="http://blazrobar.com/" class="top-banner-link">
+				<span class="top-banner-text">Awesome Free PSD<br>templates & Resources<br></span>
```
Comment (TheNightdrivingAvenger) [inline index.html]: better to use h1 instead of span

File: menu_hidden.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+nav > span {
```
Comment (TheNightdrivingAvenger) [inline menu_hidden.css]: Better to use classes instead of tags to apply styles

File: index.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+		<link rel="stylesheet" href="menu_hidden.css" media="(max-width:640px)">
+		
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+	</head>
+	<body>
+		<nav>
+			<span>Navigation</span>
+			<div class="navlist-container">
+				<ul>
+					<li><a href="PSDWebsites.html">PSD WEBSITES</a></li>
```
Comment (TheNightdrivingAvenger) [inline index.html]: How to expand a clickable area

File: styles981_1024.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
+body, html {
+	font-size: 16px;
+}
+
+.gallery-item {
+	float: left;
+	width: 33.1%;
+}
+
+main article {
```
Comment (TheNightdrivingAvenger) [inline styles981_1024.css:10]: To read: Yandex BEM (or smth the same)

**Comment by aliaksandrkrasnou** [comment]
Approved, thank you 👍

**Pull request closed without merging**